src/nyx/python/nyxus/nyxus.py

```diff
@@ -347,6 +356,32 @@ def roi_blacklist_get_summary (self):
             return None
         return s
 
+    def customize_gabor_feature (self, **kwargs):
+        """Sets parameters of feature GABOR
+
+        Keyword args:
+        * kersize (int): size of filter kernel's side. Example: customize_gabor_feature(kersize=16)
+        * gamma (float): aspect ratio of the Gaussian factor. Example: customize_gabor_feature(gamma=0.1)
+        * sig2lam (float): spatial frequency bandwidth. Example: customize_gabor_feature(sig2lam=0.8)
+        * f0 (float): frequency of the baseline lowpass filter as denominator of `\pi`. Example: customize_gabor_feature(f0=0.1)
+        * theta (float): orientation of the Gaussian in degrees 0-180. Example: customize_gabor_feature(theta=1.5708)
+        * thold (float): lower threshold of the filtered image to baseline ratio. Example: customize_gabor_feature(thold=0.025)
+        * freqs (str): comma-separated denominators of `\pi` as frequencies of Gabor filter's harmonic factor. Example: customize_gabor_feature(freqs="1,2,4,8,16,32,64")
```

Comment:
According to the feedback from Hythem, this need to be a List of numbers.

Reply:
I have a different view of utility of these string configurations, for example, an ability to add support of directional harmonics freqs="30:1,2,3;60:8,16;120:8,16;150:1,2,3" but OK, I'll rework this into a number list.